resources/assets/js/components/course/screens/Html.vue

```diff
@@ -189,50 +109,12 @@ export default {
 			}
 		},
 		addFullscreen() {
-			this.images = document.querySelectorAll(this.imagesSelector);
-
-			if (this.images.length) {
-				this.imagesLoaded = true;
-			}
+			this.images = [...this.$refs.screenHtml.querySelectorAll('img')];
```

Comment:
```suggestion
			this.images = [...this.$el.querySelectorAll('img')];
```
We don't need this `ref`.